src/landing_game_object.py

```diff
@@ -19,6 +20,7 @@ def __init__(
             pos (Vec2d, optional): 2d Position in pixel. (0,0) represents left top corner. Defaults to Vec2d().
         """
         super().__init__()
+        self.id = id
```

Comment:
Here we could catch negative numbers?

Reply:
Good idea

Reply:
Added negative number catcher in id_scope.py instead of here:
If not specified, the auto generated id will always be positive.
When using a dict with reserved ids via IDScope.set_reserved_ids(), this is the place where the check for non-negative numbers is now implemented.